index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Personal-Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="my-logo">
+        <nav class="menu-bar">
+          <a class="logo">Richard Obaseki</a>
+          <div>
+            <nav class="fas fa-bars fa_custom"></nav>
+          </div>
+        </nav>
+      </div>
+    </header>
+    <section class="headline-wrap">
+      <div class="flex headline-wrap">
+        <div class="intro-text">
+          <h1 class="introduction">
+            I’m Richard <br />Glad to see <br />
+            you!
+          </h1>
+          <p class="headline-profile margin-top-12">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hesitate to
+            contact me.
+          </p>
+          <br />
+          <a class="lets-connect margin-top-12">
+            <h3>LET’S CONNECT</h3>
+          </a>
+          <br />
+          <ul class="social-icons">
+            <li>
+              <a>
+                <img
+                class="social twitter"
+                src="Image/Twitter.png"
+                alt="Twitter icon"
+                />
+              </a>
+            </li>
+            <li>
+              <a>
+              <img
+                class="social linkedin"
+                src="Image/Linkedin.png"
+                alt="Linkedin icon"
+              />
+              </a>
+            </li>
+            <li>
+              <a>
+              <img class="social m" src="Image/Medium.png" alt="m icon" />
+              </a>
+            </li>
+            <li>
+              <a>
+              <img
+                class="social github"
+                src="Image/GitHub.png"
+                alt="github icon"
+              />
+              </a>
```

Comment:
- [ ] A suggested previously, Kindly make sure to wrap the img tag correctly with using href inside a tags as shown below.
```html
<a href="#img">
      <img class....../>
 </a>
```